tests/stubtest_allowlists/py38.txt

```diff
@@ -96,3 +96,92 @@ collections.Generator.gi_running
 collections.Generator.gi_yieldfrom
 collections.Mapping.get  # Adding None to the Union messed up mypy
 collections.Sequence.index  # Supporting None in end is not mandatory
+
+# Exists at runtime, but missing from stubs
+_dummy_thread.RLock
+_imp.source_hash
+_thread._ExceptHookArgs.n_fields
+_thread._ExceptHookArgs.n_sequence_fields
+_thread._ExceptHookArgs.n_unnamed_fields
+asyncio.BufferedProtocol.eof_received
+asyncio._enter_task
+asyncio._leave_task
+asyncio._register_task
+asyncio._unregister_task
+asyncio.protocols.BufferedProtocol.eof_received
+contextvars.ContextVar.__class_getitem__
+distutils.command.bdist_wininst
+distutils.command.bdist_wininst.bdist_wininst
+distutils.dist.DistributionMetadata.set_classifiers
+distutils.dist.DistributionMetadata.set_keywords
+distutils.dist.DistributionMetadata.set_platforms
+distutils.sysconfig.expand_makefile_vars
+distutils.sysconfig.get_python_version
+distutils.util.get_host_platform
+dummy_threading.Barrier
+dummy_threading.BoundedSemaphore
+dummy_threading.BrokenBarrierError
+dummy_threading.Condition
+dummy_threading.Event
+dummy_threading.ExceptHookArgs
+dummy_threading.Lock
+dummy_threading.RLock
+dummy_threading.Semaphore
+dummy_threading.TIMEOUT_MAX
+dummy_threading.Thread
+dummy_threading.ThreadError
+dummy_threading.Timer
+dummy_threading.active_count
+dummy_threading.current_thread
+dummy_threading.enumerate
+dummy_threading.excepthook
+dummy_threading.get_ident
+dummy_threading.local
+dummy_threading.main_thread
+dummy_threading.setprofile
+dummy_threading.settrace
+dummy_threading.stack_size
+email.headerregistry.MessageIDHeader.max_count
+ftplib.FTP.trust_server_pasv_ipv4_address
+html.parser.HTMLParser.unescape
+importlib.machinery.PathFinder.find_distributions
+importlib.metadata.EntryPoint.pattern
+importlib.util.source_hash
+lib2to3.pgen2.tokenize.COLONEQUAL
+multiprocessing.managers.SharedMemoryServer.create
+multiprocessing.managers.SharedMemoryServer.list_segments
+multiprocessing.managers.SharedMemoryServer.public
+multiprocessing.managers.SharedMemoryServer.release_segment
+multiprocessing.managers.SharedMemoryServer.shutdown
+multiprocessing.managers.SharedMemoryServer.track_segment
+plistlib.Data.asBase64
+plistlib.Data.fromBase64
+pyexpat.XMLParserType.SkippedEntityHandler
+pyexpat.XMLParserType.intern
+re.Pattern.scanner
```

Comment:
Same as in the py310 entry, I'd put this in a "Won't fix" section with a ref to #6405